### PR TITLE
Adds Luxury Bluespace Shelter Capsules, a larger and more luxurious variant.

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
add: Luxury versions of the bluespace shelter capsule are now available! Purchase them at the mining equipment vendor.
add: Cardboard cutouts have a new option: Xenomorph Maid
add: Black Carpet can now be crafted using a stack of carpet and a black crayon.
add: Black fancy tables can now be crafted using Black Carpet.
add: Shower curtains can now be recoloured with crayons, unscrewed from the floor, disassembled with wire cutters, and reassembled using cloth, plastic, and a metal rod.
/:cl:

take7

So this is what I did today.  It includes a microwave, a randomly picked three course meal, a random potted plant, two medkits, a random fake item in a display case, and a cardboard cutout of a Xenomorph Maid to keep you company during the ash storms.

Edit: New sprites for the windows and windoors, added black variant fancy tables and carpet, and adjusting the light fixtures to match the light color of the regular pods. I also rearranged the chairs and repositioned the cardboard cutout.

Edit: Carpet is less eyebleedingly yellow

Edit: Carpet matches the tables. 

Edit: This should be the final version as far as appearance changes go. There's a display case now, and the curtains are black.

### CODE REVIEW

**Comment by Armhulen** [comment]
Where's the dinner for two?

**Comment by YakumoChen** [comment]
extra perfect and extra large for that cult space base that just can't be beat

File: _maps/templates/shelter_2.dmm [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+"i" = (/obj/structure/window/reinforced/tinted/frosted,/obj/structure/toilet{icon_state = "toilet00"},/obj/machinery/light/small{tag = "icon-bulb1 (EAST)"; icon_state = "bulb1"; dir = 4; pixel_y = 10},/turf/open/floor/pod,/area/survivalpod)
+"j" = (/obj/structure/sign/mining/survival{tag = "icon-survival (EAST)"; icon_state = "survival"; dir = 4},/turf/closed/wall/mineral/titanium/survival/pod,/area/survivalpod)
+"k" = (/obj/machinery/sleeper/survival_pod,/turf/open/floor/pod,/area/survivalpod)
+"l" = (/turf/open/floor/pod,/area/survivalpod)
+"m" = (/obj/structure/window/reinforced{tag = "icon-rwindow (EAST)"; icon_state = "rwindow"; dir = 4},/turf/open/floor/pod,/area/survivalpod)
+"n" = (/turf/open/floor/carpet,/area/survivalpod)
+"o" = (/obj/structure/sink{tag = "icon-sink (EAST)"; icon_state = "sink"; dir = 4; pixel_x = 12},/turf/open/floor/carpet,/area/survivalpod)
+"p" = (/obj/structure/table/survival_pod,/obj/structure/window/reinforced,/obj/item/weapon/storage/firstaid/regular,/obj/item/weapon/storage/firstaid/brute{pixel_x = 5},/turf/open/floor/pod,/area/survivalpod)
```
Comment (YakumoChen) [inline _maps/templates/shelter_2.dmm]: coming with two free medkits borders on the line of 'way too good' unless you're covering the costs of medkits in the price too.

File: code/modules/mining/machine_vending.dm [hunk 2]
```diff
@@ -34,6 +34,7 @@
 		new /datum/data/mining_equipment("Fulton Pack",			/obj/item/weapon/extraction_pack,										1000),
 		new /datum/data/mining_equipment("Lazarus Injector",	/obj/item/weapon/lazarus_injector,										1000),
 		new /datum/data/mining_equipment("Silver Pickaxe",		/obj/item/weapon/pickaxe/silver,										1000),
+		new /datum/data/mining_equipment("Luxury Shelter Capsule",	/obj/item/weapon/survivalcapsule/luxury,							1500),
```
Comment (YakumoChen) [inline code/modules/mining/machine_vending.dm]: yeah 1500 is definitely a huge deal for two free medkits and a huge space with air, alone two brute medkits are 1200, and normal capsules are 400. I'd recommend at least 2000 or 3000 even for having a free base anywhere that's large enough to serve as a decent cult base

File: _maps/templates/shelter_2.dmm (hunk 1 above)
Comment (octareenroon91) [inline _maps/templates/shelter_2.dmm]: Two brute kits and a plain survival pod would run 1600 points separately. The value probably balances out since you have to use the 'survival pod' feature before any medkit is available.

Comment (YakumoChen) [inline _maps/templates/shelter_2.dmm]: the problem is that it's perfect for a cult to use since they can deploy it anywhere without having to build so it makes for a perfect space cube. It's large enough you could easily put a pylon there for free healing and have room for cultists to move around and convert and other such things. The normal pod is very cramped and only has like 4 or 5 useable spaces to move in, which isn't ideal for cults.

File: code/modules/mining/machine_vending.dm (hunk 2 above)
Comment (YakumoChen) [inline code/modules/mining/machine_vending.dm]: also comes with free soap (the better captain's soap even) too, which people often forget that mining can buy.

**Comment by GunHog** [comment]
Swag!

**Comment by ChangelingRain** [comment]
you didn't make new sprites aaaaaaah
I don't like it, the glass and carpet contrast too much with the pod sprites.

**Comment by Repukan** [comment]
Make it cost 5k or more.

Nothing luxurious should cost less.

**Comment by Repukan** [comment]
And points are useless past the 12 minute mark anyways.

**Comment by Bawhoppen** [comment]
Just reusing the sprites that were specifically made for the old pods takes away from the value of their use
On those grounds I have to give a :-1:

**Comment by WJohn** [comment]
I don't see the point of this except to affect more of escape. These pods aren't supposed to be lived in except for ash storms, and only up until you gain immunity via your drake armor.

**Comment by PKPenguin321** [comment]
Cute
Can the random food be robo burgers? That'd be funny
:+1: Maybe make it cost 2k though? I don't think price is a major issue really but other people seem to

**Comment by Expletive** [comment]
I'm considering all of your feedback. I'll definitely increase the price.

@ChangelingRain Okay, I'll make new sprites. Carpet is gonna be hard to do, I think. But anyway, I went and made new sprites for windoors and windows, using the pod window sprites as a guide. The windoors open from the center like the clockwork ones, and have all the proper closing/opening/emagged/denied sprites. 

newspritepreviews

File: tgstation.dme [hunk 3]
```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\basemap.dm"
+#include "_maps\templates\shelter_2.dmm"
```
Comment (coiax) [inline tgstation.dme]: Remove this

File: code/modules/mining/equipment/survival_pod.dm [hunk 4]
```diff
@@ -83,6 +83,11 @@
 	smooth = SMOOTH_MORE
 	canSmoothWith = list(/turf/closed/wall/mineral/titanium/survival, /obj/machinery/door/airlock/survival_pod, /obj/structure/window/shuttle/survival_pod)
 
+/obj/structure/window/reinforced/survival_pod
+	name = "pod window"
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "pwindow"
```
Comment (ChangelingRain) [inline code/modules/mining/equipment/survival_pod.dm]: I don't believe windows handle diagonal directions properly, you might need to test that out for the like, one diagonal window corner you're adding here.

File: code/game/objects/structures/tables_racks.dm [hunk 5]
```diff
@@ -271,12 +271,18 @@
...
 
 /obj/structure/table/wood/fancy/New()
 	icon = 'icons/obj/smooth_structures/fancy_table.dmi' //so that the tables place correctly in the map editor
 	..()
 
+/obj/structure/table/wood/fancy/black
+	icon_state = "fancy_table_black"
+
+/obj/structure/table/wood/fancy/black/New()
+	icon = 'icons/obj/smooth_structures/fancy_table_black.dmi'
```
Comment (ChangelingRain) [inline code/game/objects/structures/tables_racks.dm]: ..() directly above this

**Comment by ChangelingRain** [comment]
I think the yellow carpet edges stand out a lot, you might want to darken them to match the fancy tables?
but yeah this is a lot better

**Comment by ChangelingRain** [comment]
oh right can you put the tubes fluff object below the bed

**Comment by Expletive** [comment]
@ChangelingRain  Tested it out, and the one diagonal window doesn't block movement, so it's fine for prettying up the place.

**Comment by Kromgar** [comment]
Thanks for the new cult base brah

**Comment by ChangelingRain** [comment]
My concern was more if it prevented clicks/airflow, since fulltile windows use a diagonal dir to cheat and act as normal objects.

**Comment by Armhulen** [comment]
My concern is that carpet, EUGH.

**Comment by Expletive** [comment]
@ChangelingRain It does indeed seem to block clicks. Setting density = 0 on it seems to fix it.

**Comment by ChangelingRain** [comment]
You might want to reposition the sink/toilet since the sink is layering over the toilet

**Comment by lzimann** [comment]
remove maid

File: code/modules/mining/equipment/survival_pod.dm [hunk 6]
```diff
@@ -297,3 +297,38 @@
...
+						/obj/item/weapon/katana/energy,
+						/obj/item/weapon/hierophant_club,
+						/obj/item/weapon/his_grace,
+						/obj/item/weapon/gun/ballistic/minigun,
+						/obj/item/weapon/gun/ballistic/automatic/l6_saw,
+						/obj/item/weapon/gun/magic/staff/chaos,
+						/obj/item/weapon/gun/magic/staff/spellblade,
+						/obj/item/weapon/gun/magic/wand/death,
+						/obj/item/weapon/gun/magic/wand/fireball,
+						/obj/item/weapon/disk/nuclear,
```
Comment (octareenroon91) [inline code/modules/mining/equipment/survival_pod.dm]: Are perfect replicas of the nuke disk allowed?

Comment (lzimann) [inline code/modules/mining/equipment/survival_pod.dm]: No, this shouldn't have a nuke disk.

Comment (Expletive) [inline code/modules/mining/equipment/survival_pod.dm]: I have no idea.

File: code/modules/mining/equipment/survival_pod.dm [hunk 7]
```diff
@@ -297,3 +297,38 @@
 	anchored = TRUE
 	layer = BELOW_MOB_LAYER
 	density = FALSE
+
+/obj/item/fakeartefact
```
Comment (ChangelingRain) [inline code/modules/mining/equipment/survival_pod.dm]: baaaad

Comment (Expletive) [inline code/modules/mining/equipment/survival_pod.dm]: Okay. I can change it to create real versions?

**Comment by ChangelingRain** [comment]
You new new new carpet still doesn't smooth properly; you can't flip or rotate a grid pattern like this one

**Comment by Vivalas** [comment]
The first picture looked better than the one with new sprites imo.

File: code/game/objects/structures/watercloset.dm [hunk 8]
```diff
@@ -106,13 +105,18 @@
...
+	var/obj/item/secret
+	var/secret_type = null
+
+/obj/structure/toilet/secret/Initialize(mapload)
+	. = ..()
+	if (secret_type)
+		secret = new secret_type(src)
+		secret.desc += " It's a secret!"
+		w_items += secret.w_class
+		contents += secret
```
Comment (lzimann) [inline code/game/objects/structures/watercloset.dm]: Manually setting contents is not needed here. Doing `new secret_type(src)` already makes sure it's inside the toilet.

Comment (Expletive) [inline code/game/objects/structures/watercloset.dm]: Oh, alright. Thanks.

**Comment by AnturK** [comment]
Seems like you reverted the mining mob file split.

**Comment by Expletive** [comment]
@AnturK Had an accident with git. I didn't have master checked out when I tried to merge upstream, and it broke the build, so I ended up reverting to try and fix it.

**Comment by FantasticFwoosh** [comment]
Here is a alternative design i think you might like using a lot of the same elements. Your new sprites look pretty and i do think they would convey over to my alternative design just as well. No use for a windoor in it though.

got some rough edges because im not the best with map maker

**Comment by Kraseo** [comment]
I like expletive's design better

**Comment by Kromgar** [comment]
Do the display cases persist cross rounds?

**Comment by octareenroon91** [comment]
>persist?

It looks like only the trophy subtype persists.

File: code/game/objects/structures/watercloset.dm [hunk 9]
```diff
@@ -580,7 +580,7 @@
 /obj/structure/curtain/update_icon()
 	if(!open)
 		icon_state = "closed"
-		layer = WALL_OBJ_LAYER
+		layer = FLY_LAYER //Changed from WALL_OBJ_LAYER so that when a shower is being used, the curtain is in front of the water droplets but behind the steam.
```
Comment (ChangelingRain) [inline code/game/objects/structures/watercloset.dm]: I think you should change the steam's layer instead

File: code/game/objects/structures/watercloset.dm (hunk 8 above)
Comment (ChangelingRain) [inline code/game/objects/structures/watercloset.dm]: still gotta do this!

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 10]
```diff
@@ -0,0 +1,1143 @@
+/mob/living/simple_animal/hostile/asteroid
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: you uh.... gotta fix this

File: html/changelogs/.all_changelog.yml [hunk 11]
```diff
@@ -11530,11 +11530,6 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   Steelpoint:
   - rscadd: A New Iron Hawk troop transport ruin has been added to lavaland. Can the
       sole surviving Marine somehow survive the horrors of lavaland? Lore fluff included.
-  - rscadd: Central Command has listened to complaints and, as such, has now stationed
-      "real" Private Security Officers at centcom docks.
-  - rscadd: A new Nanotrasen Security Officer NPC variant is available to admins,
-      this 'peaceful' version will only attack people who attack it first. Great for
```
Comment (ChangelingRain) [inline html/changelogs/.all_changelog.yml]: you kind of have a problem here

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm (hunk 10 above)
Comment (Tacolizard) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: wat

File: html/changelogs/.all_changelog.yml (hunk 11 above)
Comment (Steelpoint) [inline html/changelogs/.all_changelog.yml]: Someone dislikes my change

**Comment by Steelpoint** [comment]
Man this PR is accidently reverting a lot of changes.

I would suggest just closing this and starting fresh.

**Comment by Expletive** [comment]
How would I fix it? I accidentally merged something that broke the build. Was reverting the thing not the way to do it?

**Comment by Expletive** [comment]
Okay, I'll make a new one, since this one is broken.

**Comment by Repukan** [comment]
THE DREAM IS FUCKING DEAD
